integ-forum/integ-forum-social/src/main/java/org/exoplatform/forum/ext/impl/ActivityUtils.java

```diff
@@ -38,10 +41,23 @@ public static String processContent(String content) {
   }
   
   public static String getSpaceGroupId(String categoryId) {
+    String categoryPath = "";
```

Comment:
Remove line 44 and move it to line 51: String categoryPath = category.getPath();